### ISSUE TITLE
Provide health-measures per index

### ISSUE THREAD

**Issue by centic9** [dynaTrace/Dynatrace-Elasticsearch-Plugin#2]
Migrated from #1, originally submitted by user @keertisurapaneni

I have a request for the ES fastpack you are developing. Is there a way you can create measures for different indices (like index name, doc count, delete count, health, status, pri, rep, store size, etc..)
The below command will pull those values:
curl -XGET http://chdd-es01.cps.local:9200/_cat/indices?v

That way we can create a dashbord for index health. The dashboard can have tables for each measure (doc count, delete count, health, etc..) 

For ex: We can have a dashlet table for measure 'document count' which would have splittings by index name.

below is the command to extract doc count for each index:
curl -XGET http://chdd-es01.cps.local:9200/_cat/indices?v | awk "{print $3, $6}"

example output:
index docs.count
unilog_04 99377
cps_documents 495303
customer_alias-20160331 160917

**Comment by centic9** [comment]
We can add this, but I likely would not enable this by default, but rather add a switch to the plugin-configuration to only fetch and provide these measures when the user decides that they are useful for him/her. 

Reasons are:
- The indices-stats can become a bit costly if you have a large number of indexes
- If you have different index-names and create/delete many indices constantly, you can cause trouble in Dynatrace if the splitting-names constantly change and thus you end up with a huge number of unique splittings which Dynatrace still needs to maintain over a long period of time

**Comment by keertisurapaneni** [comment]
We have 42 indices as of now. Is that a lot?

**Comment by centic9** [comment]
No, that is not a lot, Elasticsearch can easily handle hundreds, maybe up to a few thousand, depending on complexity of mappings. 

But I am speaking for the general case in the plugin where we need to avoid disrupting other peoples environments by simply adding this in a too naive way.

With a switch you can easily enable it for you and everybody else who wants this level of detail can as well, but it will not affect others immediately who do not need it.

**Comment by keertisurapaneni** [comment]
Hi Dominik,

You are right. Switch is a brilliant idea. People who need it can switch it on. I am not a coding person. Will you be able to  work on this and add a switch? 

Thank you,
Keerti

**Comment by centic9** [comment]
I'll try to squeeze it in eventually when I revisit the plugin for things that we need ourselves, but I cannot work on this just now...

**Comment by keertisurapaneni** [comment]
Sure, thanks a lot Dominik. Appreciate it.
